### PR TITLE
[1LP][RFR] New Test: Testing CRUD operations on imported domains

### PR DESCRIPTION
**Pull request by ganeshhubale**
Signed-off-by: Ganesh Hubale <ghubale@redhat.com>

## Purpose or Intent
- Testing crud operation on imported domains with respect to different sources
### PRT Run
{{ pytest: cfme/tests/automate/test_file_import.py::test_crud_imported_domains --use-template-cache -qsvv }}

### CODE REVIEW

File: cfme/tests/automate/test_file_import.py [hunk 1]
```diff
@@ -31,3 +36,68 @@ def test_domain_import_file(import_datastore, import_data):
...
+    if import_data.file_name == "bz_1753586_user.zip":
+        with update(domain):
+            domain.description = fauxfactory.gen_alpha()
+        domain.delete()
+        view.flash.assert_message(f'Automate Domain "{domain.description}": Delete successful')
+    elif import_data.file_name == "bz_1753586_user_locked.zip":
+        with update(domain):
+            domain.description = fauxfactory.gen_alpha()
+        domain.delete()
+        view.flash.assert_message(f'Automate Domain "{domain.description}": Delete successful')
```
Comment (john-dupuy) [inline cfme/tests/automate/test_file_import.py]: @ganeshhubale Since these `if, elif` blocks are doing the same thing, you can combine them.

Comment (ganeshhubale) [reply cfme/tests/automate/test_file_import.py]: My bad!
Thanks @john-dupuy :+1:

File: cfme/tests/automate/test_file_import.py [hunk 2]
```diff
@@ -90,3 +92,63 @@ def test_upload_blank_file(appliance, upload_file):
...
+            3. From within the new Domain_Test02 dir, edit the __domain__.yaml file so that the
+               'name' attribute equals Domain_Test02
+               NOTE: Do not change the 'source' attribute from 'system' to 'user' this time.
+            4. Archived the new Domain_Test02 dir and upload to CloudForms Appliance
+        expectedResults:
+            1.
+            2.
+            3.
+            4. The imported domains should be Custom Domains or these imported domains should gets
+               deleted from db
```
Comment (valaparthvi) [inline cfme/tests/automate/test_file_import.py]: The test case is simple enough and I really don't think we need testSteps, besides testSteps doesn't seem to be related to what your test case is actually doing there. I'd rather the testSteps be removed, they are a little confusing. Let me know what you think.

Comment (ganeshhubale) [reply cfme/tests/automate/test_file_import.py]: Removed testSteps and expectedResults.

File: cfme/automate/explorer/domain.py [hunk 3]
```diff
@@ -292,13 +286,6 @@ def update(self, updates):
         view = self.create_view(DomainDetailsView, override=updates)
         assert view.is_displayed
         view.flash.assert_no_error()
-        if changed:
-            text = (updates.get('description', self.description)
-                    or updates.get('name', self.name))
-            view.flash.assert_message('Automate Domain "{}" was saved'.format(text))
```
Comment (jawatts) [inline cfme/automate/explorer/domain.py]: **Required** agree with removing this in this module but I think there should be a test checking these messages flash messages.

Comment (ganeshhubale) [reply cfme/automate/explorer/domain.py]: @jawatts 
I agree. I am thinking of updating testcase ```test_domain_crud``` to check domain delete flash message.
